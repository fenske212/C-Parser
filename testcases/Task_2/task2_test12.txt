X043eSc -> Y043eSc *
Y043eSc -> e043eSc Y043eSc *
Y043eSc -> *
Z043eSc -> b043eSc Z043eSc c043eSc *
Z043eSc -> a043eSc *
A043eSc -> b043eSc c043eSc a043eSc *
B043eSc -> c043eSc B043eSc *
B043eSc -> d043eSc *
C043eSc -> e043eSc C043eSc a043eSc *
C043eSc -> *
#

