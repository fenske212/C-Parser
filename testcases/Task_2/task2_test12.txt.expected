X043eSc -> Y043eSc 
Y043eSc -> e043eSc Y043eSc 
Y043eSc -> # 
